tests/io_test.py

```diff
@@ -238,13 +235,80 @@ def testGetDataSets(self):
         :raise: AssertionError if the input and returned dataset names don't match
         '''
         self._create_file(columns=self.dict_columns, 
-                          path_prefix='{}/iotest_dict_columns'.format(IOTest.io_test_dir))     
+                          prefix_path='{}/iotest_dict_columns'.format(IOTest.io_test_dir))     
         datasets = ak.get_datasets('{}/iotest_dict_columns_LOCALE0'.format(IOTest.io_test_dir))
 
         self.assertEqual(3, len(datasets)) 
         for dataset in datasets:
             self.assertIn(dataset, self.names)
 
+    def testSaveStringsDataset(self):
+        # Create, save, and load Strings dataset
+        strings_array = ak.array(['string {}'.format(num) for num in list(range(1,11))])
+        strings_array.save('{}/strings-test'.format(IOTest.io_test_dir), dataset='strings')
+        r_strings_array = ak.load_all('{}/strings-test'.format(IOTest.io_test_dir))['strings']
```

Comment:
This line should use `ak.load()`, not `ak.load_all()`, because the goal is to verify that `load` is the inverse of `save`.

Reply:
Gotcha, fixed.